scout/adapter/mongo/panel.py

```diff
@@ -297,6 +297,7 @@ def gene_to_panels(self, case_obj):
                         panel_name, panel_version
                     )
                 )
+                continue
```

Comment:
You got to love the comment before.. "Raise exception here???". Codefactor should have called us out on it.

Reply:
I've missed it completely, let's remove it!